workflows/kfdrc-single-sample-genotyping-wf.cwl

```diff
@@ -121,138 +159,146 @@ outputs:
   peddy_html: {type: 'File[]', doc: 'html summary of peddy results', outputSource: peddy/output_html}
   peddy_csv: {type: 'File[]', doc: 'csv details of peddy results', outputSource: peddy/output_csv}
   peddy_ped: {type: 'File[]', doc: 'ped format summary of peddy results', outputSource: peddy/output_peddy}
+  hardfiltered_vcf: {type: 'File?', secondaryFiles: [{pattern: '.tbi', required: true}], outputSource: gatk_hardfiltering/hardfiltered_vcf}
   vep_annotated_vcf: {type: 'File[]', outputSource: annotate_vcf/annotated_vcf}
 
 steps:
+  filtering_defaults:
+    run: ../tools/filtering_defaults.cwl
+    in:
+      num_vcfs:
+        source: input_vcfs
+        valueFrom: $(self.length)
+      experiment_type: experiment_type
+    out: [low_data, snp_tranches, indel_tranches, snp_annotations, indel_annotations, snp_ts_filter_level, indel_ts_filter_level, snp_hardfilter, indel_hardfilter]
   dynamicallycombineintervals:
     run: ../tools/script_dynamicallycombineintervals.cwl
-    doc: 'Merge interval lists based on number of gVCF inputs'
+    hints:
+    - class: 'sbg:AWSInstanceType'
+      value: c5.9xlarge
     in:
       input_vcfs: input_vcfs
       interval: unpadded_intervals_file
     out: [out_intervals]
-  gatk_import_genotype_filtergvcf_merge:
-    run: ../tools/gatk_import_genotype_filtergvcf_merge.cwl
+  gatk_genomicsdbimport_genotypegvcfs:
+    run: ../tools/gatk_genomicsdbimport_genotypegvcfs.cwl
     hints:
     - class: 'sbg:AWSInstanceType'
-      value: r4.4xlarge;ebs-gp2;500
-    doc: 'Use GATK GenomicsDBImport, VariantFiltration GenotypeGVCFs, and picard MakeSitesOnlyVcf to genotype, filter and merge gVCF
-      based on known sites'
+      value: c5.9xlarge
```

Comment:
Oh my, what an ancient instance type request we had before....

Reply:
True and a bad instance family request. It's a 2-1 RAM-CPU ratio so it should have been on the `c` family.